User/Library/UserMisc.c

```diff
@@ -672,3 +674,23 @@ DevicePathFromHandle (
   return NULL;
 }
 
+VOID
+EFIAPI
+Sha512TransformAvx (
```

Comment:
Move these to `OcCryptoLib/Sha512AvxDummy.c` and remove `Ia32/Sha512AvxDummy.nasm`. Then update `User/Makefile` to include `Sha512AvxDummy.o` in:

```
	#
	# OcCryptoLib targets.
	#
	OBJS    += RsaDigitalSign.o BigNumMontgomery.o BigNumPrimitives.o BigNumWordMul64.o Sha2.o SecureMem.o
```